packages/google_maps_flutter/google_maps_flutter/lib/src/controller.dart

```diff
@@ -64,6 +64,14 @@ class GoogleMapController {
       case 'marker#onTap':
         _googleMapState.onMarkerTap(call.arguments['markerId']);
         break;
+      case 'marker#onDrag':
+        _googleMapState.onMarkerDrag(call.arguments['markerId'],
+            LatLng._fromJson(call.arguments['position']));
+        break;
+      case 'marker#onDragStart':
+        _googleMapState.onMarkerDragStart(call.arguments['markerId'],
+            LatLng._fromJson(call.arguments['position']));
+        break;
```

Comment:
This logic has moved to the `google_maps_flutter_platform_interface` package, [here](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart#L118).

These are now defined as a Stream of the appropriate Marker*Event (MarkerDragEvent, MarkerDragStartEvent), [like so](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart#L88-L91). Also, these streams are part of the platform interface class, [here](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart#L277-L280).